CORE_CONTRIBUTOR.md

```diff
@@ -7,20 +7,20 @@ We believe that our most valuable resource is the passionate community members w
 ## How does one become a core contributor?
 Contributors who have displayed lasting commitment to the evolution and maintenance of _fastlane_ will be invited to become core contributors. For instance, contributors who:
 - Love to help out other users with issues on GitHub
-- Continue to make _fastlane_ a stable product and encourage features aligned with the _fastlane_ [vision](https://github.com/fastlane/fastlane/blob/master/VISION.md)
+- Continue to make _fastlane_ a stable product and encourage features aligned with [_fastlane_'s vision](https://github.com/fastlane/fastlane/blob/master/VISION.md)
```

Comment:
Don't mean to be picky here, but we have three instances of using `_fastlane_ XX`: 
`_fastlane_'s vision` twice and `the [_fastlane_ Code of Conduct]`... I think we should use `the _fastlane_ XX` in all places instead of using an apostrophe, but you can take or leave that suggestion for sure!